.github/workflows/build-lint-test.yml

```diff
@@ -8,16 +8,30 @@ on:
 
 env:
   CARGO_TERM_COLOR: always
-
+  ASSET_DIR: ${{ github.workspace }}/frontend/dist
 jobs:
   backend:
     name: Backend
+    needs:
+        - frontend
```

Comment:
Now we also don't need this dependency any more

Reply:
Fixed in [f12b881](https://github.com/kiesraad/abacus/pull/429/commits/f12b8814300aa10e2ea17fd1553b235d422f30dc)